workspaces/jenkins/plugins/jenkins/package.json

```diff
@@ -12,9 +12,22 @@
     ]
   },
   "publishConfig": {
-    "access": "public",
-    "main": "dist/index.esm.js",
-    "types": "dist/index.d.ts"
```

Comment:
I'm pretty sure we still need these main and types entries.

Reply:
Seems it's fine looking at how we did it for GitHub Actions: https://github.com/backstage/community-plugins/blob/main/workspaces/github-actions/plugins/github-actions/package.json